docs/using.rst

```diff
@@ -28,6 +28,12 @@ anywhere.
    https://importlib-metadata.readthedocs.io/
       The documentation for ``importlib_metadata``, which supplies a
       backport of ``importlib.metadata``.
+      This includes an `API reference
+      <https://importlib-metadata.readthedocs.io/en/latest/api.html>`__
+      for this module's classes and functions,
+      as well as a `migration guide
+      <https://importlib-metadata.readthedocs.io/en/latest/migration.html>`__
+      for existing users of ``pkg_resources``.
```

Comment:
I'm a little concerned about adding multiple references and keeping those in sync. Why not just "..., which supplies a backport of ``importlib.metadata`` and includes an API reference and migration guide for users of pkg_resources." That would inform the users of the presence of those aspects without keeping them linked.

Reply:
That's a good point. We could use `:ref:` via Intersphinx which would keep them in sync automatically, though that would require adding intersphinx to the CPython docs build config which I'm not sure the downstreams would like (since it requires either network access or a local cache when building docs). But if you think so, we can just go with it for now—you're the maintainer, of course.